zuko-ui/src/components/Article/Article.jsx

```diff
@@ -11,7 +11,7 @@ function Article({data = {}}){
             {data?.categories ? 
             <div className={styles.categories}>
                 {data?.categories.map((el, i) => (
-                    <span className={styles.subtitle}>{el?.name}</span>
+                    <span className={styles.subtitle} key={i}>{el?.name}</span>
```

Comment:
Let's try not to use index of elements as keys as React will then always render the whole list let's try to have the keys as unique identifiers so it can swap just the items that have been changed